src/components/AddBook.js

```diff
@@ -1,29 +1,57 @@
-const AddBook = () => (
-  <div className="add-book-section">
-    <div className="add-book-header">ADD NEW BOOK</div>
-    <form className="add-book-form">
-      <input
-        type="text"
-        className="add-book-title"
-        placeholder="Title"
-        name="title"
-      />
-      <input
-        type="text"
-        className="add-book-author"
-        placeholder="Author"
-        name="author"
-      />
-      <div className="add-book-wrapper">
-        <select name="category" className="add-book-category">
-          <option value="category">Category</option>
-          <option value="fantasy">Fantasy</option>
-          <option value="fantasy">Journalism</option>
-        </select>
-        <button type="button" className="btn add-book-btn">ADD BOOK</button>
-      </div>
-    </form>
-  </div>
-);
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
+  const getElementById = (id) => document.getElementById(id);
+
```

Comment:
**Changes Required**
- [ ] The component should contain: - A local state with `title`, `author`. Every time a user changes the input value of either `title` or `author`, react should re-render in response to the state update. 
You can then pass the value of title and author to the function handling onSubmit to dispatch the action to the store.

Reply:
Done! Thank you - great review.